Modules/modSCC.ACM

```diff
@@ -3,9 +3,12 @@ Option Explicit
 
 Public Enum SccSystem
   sccSystemSubversion = 1  ' Supported
+  sccSystemText = 4        ' Supported
+  
+  ' These are placeholders for future possible enhanced support
+  ' for specific SCC systems

```

Comment:
```suggestion
  ' for specific SCC systems. The Text implementation does work generically
  ' with GIT but does not integrate fully, ie. specific git functions (commit, push, etc)
  ' have to be performed by the user
```

Reply:
I have added a comment based off of this suggestion